frontend/src/patientApp/timeOfTest/TestResult.stories.tsx

```diff
@@ -0,0 +1,197 @@
+import { Meta } from "@storybook/react";
+import { cloneDeep } from "lodash";
+import createMockStore from "redux-mock-store";
+import { Provider } from "react-redux";
+
+import Page from "../../app/commonComponents/Page/Page";
+import PatientHeader from "../PatientHeader";
+import { VerifyV2Response } from "../PxpApiService";
+
+import TestResult from "./TestResult";
+
+const mockStore = createMockStore([]);
+
+export default {
+  title: "Pxp Test Results",
+  component: TestResult,
+  parameters: {
+    layout: "fullscreen",
+  },
+  argTypes: {},
+  args: {},
+} as Meta;
+
+const data = {
+  testResult: {
+    testEventId: "12321312",
+    result: "POSITIVE" as TestResult,
+    results: [
+      { disease: { name: "COVID-19" }, result: "POSITIVE" },
+    ] as PxpMultiplexResult[],
+    dateTested: new Date("2021-08-20").toDateString(),
+    correctionStatus: "ORIGINAL",
+    deviceType: {
+      name: "Fake device",
+      model: "Fake model",
+    },
+    organization: {
+      name: "Fake organization",
+    },
+    facility: {
+      name: "Facility Name",
+      cliaNumber: "12D4567890",
+      street: "555 Fake St",
+      streetTwo: "APT 1",
+      state: "NC",
+      city: "Raleigh",
+      zipCode: "27601",
+      phone: "6318675309",
+      orderingProvider: {
+        firstName: "Ordering",
+        lastName: "Provider",
+        middleName: "OP Middle Name",
+        npi: "fake-npi",
+      },
+    },
+    patient: {
+      firstName: "First",
+      middleName: "Middle",
+      lastName: "Last",
+      birthDate: "08/07/1990",
+    },
+  } as VerifyV2Response,
+};
+
+export const PositiveCovid = () => {
+  let store = mockStore(data);
+  return (
+    <Provider store={store}>
+      <Page>
+        <PatientHeader />
+        <TestResult />
+      </Page>
+    </Provider>
+  );
+};
+
+export const NegativeCovid = () => {
+  let editedData = cloneDeep(data);
+  editedData.testResult.results = [
+    { disease: { name: "COVID-19" }, result: "NEGATIVE" },
+  ];
+  let store = mockStore(editedData);
+  return (
+    <Provider store={store}>
+      <Page>
+        <PatientHeader />
+        <TestResult />
+      </Page>
+    </Provider>
+  );
+};
+
+export const InconclusiveCovid = () => {
+  let editedData = cloneDeep(data);
+  editedData.testResult.results = [
+    { disease: { name: "COVID-19" }, result: "UNDETERMINED" },
+  ];
+  let store = mockStore(editedData);
+  return (
+    <Provider store={store}>
+      <Page>
+        <PatientHeader />
+        <TestResult />
+      </Page>
+    </Provider>
+  );
+};
+
+export const PositiveFluMultiplex = () => {
```

Comment:
⚠️ to see the multiplex ones locally run `REACT_APP_MULTIPLEX_ENABLED=true yarn storybook` in the `frontend` directory.